CHANGES.md

```diff
@@ -6,6 +6,7 @@
     - This requires cargo web of >= 0.6.23, use `cargo install -f cargo-web` to update
 - Add the ability to take screenshots of the window or surface with `Window::screenshot`
 - Fix a bug in web key input where any key past F15 would not map correctly
+- Add the ability to use a custom initializer for `State` implementors using `run_with`
```

Comment:
It looks like you branched off master so your changelog entry isn't quite right.